lib/twingly/livefeed/version.rb

```diff
@@ -0,0 +1,5 @@
+module Twingly
+  module LiveFeed
+    VERSION = "5.0.1"
```

Comment:
> Yes, not sure what's "best" though.

If there are no difference in dependencies between Search and LiveFeed code, then I think it's best to have it as one gem. Easier for those who want to use both, for some reason. And shouldn't incur any burden on those who only use one API.